reporting/sending_to_dhis2.md

```diff
@@ -0,0 +1,20 @@
+### Sending Reports to DHIS2
+
+1. Set the DHIS2 UUID for the facility that is sending the data by:
+
+   * Got to Legacy System Administration-&gt;Mantainance-&gt;Settings-&gt;Ugandaemr.
+     > ![](../assets/facilityUUID.PNG)
+
+2. Set the DHIS2 Server URL, username and password by:
+
+   * Got to Legacy System Administration-&gt;Mantainance-&gt;Settings-&gt;Ugandaemrsync
+
+   * Save the Above settings
+
+3. Download the Section 4 HTS Report in JSON Format
+
+4. Click the SendToDHIS2 action item on the reports page
+
+5. On the Preview window, confirm the values displayed and Click the SendToDHIS2 button
+
+6. Wait for response message from the server. If the report is sent you will receive a message **"Report Has Been Successfully Sent"**
```

Comment:
@slubwama Can you add troubleshooting tips 

- Connection Error when there is no Internet
- Error from DHIS2 
- Generating the report then sending it later